crates/net/network/src/config.rs

```diff
@@ -121,15 +122,19 @@ impl<C> NetworkConfig<C> {
     ) -> Self {
         let rlpx_port = self.listener_addr.port();
         let chain = self.chain_spec.chain;
-        let fork_id = self.status.forkid;
+        let fork_id = self.chain_spec.latest_fork_id();
         let boot_nodes = self.boot_nodes.clone();
 
         let mut builder =
             reth_discv5::Config::builder(rlpx_port).add_unsigned_boot_nodes(boot_nodes.into_iter());
 
-        if chain.is_optimism() {
-            builder = builder.fork(OPSTACK, fork_id)
+        if chain.named() == Some(NamedChain::Mainnet) {
+            builder = builder.fork(network_key::ETH, fork_id)
         }
+        // todo: set op EL fork id
+        /*if chain.is_optimism() {
+            builder = builder.fork(network_key::, fork_id)
+        }*/
```

Comment:
i assume this needs to be handled before merge?

Reply:
nope, it's left blank for now. OP haven't decided on a network key yet. `'opstack'` is reserved for cl. so no fork id on op el enrs for now.